contracts/RealmDiamond/facets/RealmFacet.sol

```diff
@@ -194,43 +243,80 @@ contract RealmFacet is Modifiers {
     return s.parcels[_realmId].reservoirCapacity;
   }
 
-  function getHumbleGrid(uint256 _parcelId) external view returns (uint256[8][8] memory output_) {
+  function getHumbleGrid(uint256 _parcelId, uint256 _gridType) external view returns (uint256[8][8] memory output_) {
     require(s.parcels[_parcelId].size == 0, "RealmFacet: Not humble");
-    for (uint256 i; i < 8; i++) {
-      for (uint256 j; j < 8; j++) {
-        output_[i][j] = s.parcels[_parcelId].buildGrid[i][j];
+    if (_gridType == 0) {
+      for (uint256 i; i < 8; i++) {
+        for (uint256 j; j < 8; j++) {
+          output_[i][j] = s.parcels[_parcelId].buildGrid[i][j];
+        }
+      }
+    } else if (_gridType == 1) {
+      for (uint256 i; i < 8; i++) {
+        for (uint256 j; j < 8; j++) {
+          output_[i][j] = s.parcels[_parcelId].tileGrid[i][j];
+        }
```

Comment:
Can combine the two `for` statements and include the conditional inside the for loop, to reduce code